### PR TITLE
Enable Credhub co-located deployment for development BOSH

### PR DESCRIPTION
**Pull request by jontours**
Testing on development.

### CODE REVIEW

File: operations/external-db.yml [hunk 1]
```diff
@@ -46,5 +46,16 @@
...
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=credhub/properties/credhub/data_storage
+  value:
+    type: postgres
+    host: ((terraform_outputs.credhub_rds_host))
+    port: ((terraform_outputs.credhub_rds_port))
+    username: ((terraform_outputs.credhub_rds_username))
+    password: ((terraform_outputs.credhub_rds_password))
+    database: credhub
+    require_tls: false
```
[the remark this answers is not in the document]
Comment (jontours) [reply operations/external-db.yml]: Ah, good catch.

[the remark this answers is not in the document]
Comment (rogeruiz) [reply operations/external-db.yml]: yes please

[the remark this answers is not in the document]
Comment (rogeruiz) [reply operations/external-db.yml]: addressed with cba8ca2

**Comment by rogeruiz** [comment]
Looks like TLS for CredHub is requiring the private key, cert, and CA. I’m gonna follow these docs, https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html and troubleshoot some more. Either way. I gotta verify that SSL is forced on the RDS instance too.

File: operations/external-db.yml [hunk 2]
```diff
@@ -46,5 +46,115 @@
...
+    type: postgres
+    host: ((terraform_outputs.credhub_rds_host))
+    port: ((terraform_outputs.credhub_rds_port))
+    username: ((terraform_outputs.credhub_rds_username))
+    password: ((terraform_outputs.credhub_rds_password))
+    database: credhub
+    require_tls: true
+
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=credhub/properties/credhub/tls?
```
Comment (jmcarp) [inline operations/external-db.yml]: Should this be https://github.com/pivotal-cf/credhub-release/blob/master/jobs/credhub/spec#L193-L194 ?

**Comment by rogeruiz** [comment]
This now includes all environments set up with secrets for CredHub.

File: ci/pipeline.yml [hunk 3]
```diff
@@ -171,8 +172,26 @@ jobs:
...
   - put: stagingbosh-deployment
-    params:
-      <<: *bosh-deployment
+    params: &bosh-deployment
+      manifest: bosh-deployment/bosh.yml
+      stemcells:
+      - bosh-stemcell/*.tgz
+      ops_files:
+      - bosh-deployment/uaa.yml
+      - bosh-deployment/aws/cpi.yml
```
Comment (jontours) [inline ci/pipeline.yml]: We still need credhub.yml here, ditto for other bosh(es).

**Pull request merged**